en/smart_batteries/README.md

```diff
@@ -0,0 +1,13 @@
+# Smart Batteries
+
+Smart Batteries keep track of the state of the battery, including its exact state of charge, cell voltages, time-to-empty, and temperature readings. This allows for easier flight planning and more accurate prediction of failure conditions: [Safety > Low Battery Failsafe](../config/safety.md#low-battery-failsafe).
+
+
+
+:::note
```

Comment:
This is the driver for a battery connected to the SMBUS right (i.e. I2C)? That's just one way to connect a battery - for example, we often recommend MAVLink.

Can you please change this from a note to a new section at the very end

---

## Further Information

- [batt_smbus](../modules/modules_driver.md) - PX4 SMBus Battery Driver docs
- [MAVLink Battery Protocol](https://mavlink.io/en/services/battery.html)